lib_eio_linux/eio_linux.mli

```diff
@@ -248,4 +248,29 @@ module Low_level : sig
   (** [getaddrinfo host] returns a list of IP addresses for [host]. [host] is either a domain name or
       an ipaddress. *)
 
+  module Process : sig
+    type t
+    (** A subprocess *)
+
+    val spawn :
+      ?env:Spawn.Env.t ->
+      ?cwd:Spawn.Working_dir.t ->
+      ?stdin:FD.t ->
+      ?stdout:FD.t ->
+      ?stderr:FD.t ->
+      sw:Switch.t ->
+      string ->
+      string list ->
+      t
+    (** Spawns a subprocess. By default all of the optional arguments are inherited from the
+        calling process. If the process has not finished when the switch is released, the process
+        will be sent [Sys.sigkill]. *)
+
+    val await_exit : t -> Unix.process_status
```

Comment:
why not just call this `wait`? It's shorter and has precedent in *wait(2)*.

Reply:
Sounds good to me, done in https://github.com/ocaml-multicore/eio/pull/435/commits/944664cfd161d60d325ab293d801a8cbd653f4b3 (I think it was named that way initially because I exposed the status as a promise to the user, hence `await`).

Reply:
We don't want to call `wait(2)` here, though. The process should get reaped whether the users calls this or not. This function should just be awaiting the promise.